doc-assets/node/18005-search-videos/18005.html

```diff
@@ -193,15 +90,14 @@ <h2 id="Basic_search">Basic search</h2>
     <div class="bcls-footnote" id="note1-1"><sup><strong>[1-1]</strong></sup> Note: searching by id is included for consistency, but a search on <code>q=id:12345</code> will return exactly the same results as the request <code>https://cms.api.brightcove.com/v1/accounts/{account_id}/videos/12345</code></div>
 
     <div class="bcls-footnote" id="note1-2"><sup><strong>[1-2]</strong></sup> If you have a list-type custom field and want to return videos that have one of several values, you can do so like this:
-
       <p>Let's say you have a field called <code>color</code> that can take the values: <code>red</code>, <code>green</code>, <code>yellow</code>, or <code>blue</code>. You want to find videos that have that field set to the value <code>green</code> or <code>blue</code>:</p>
 
-<pre> <code class="language-http">q=color:green+color:blue</code></pre>
+      <pre> <code class="language-http">q=color:green%20color:blue</code></pre>
```

Comment:
All example queries now exclusively use `%20` as a term delimiter

Reply:
I'm ambivalent on this -- doesn't this make it look like our syntax is harder to use than it is?

Reply:
I sort of think it does too, but depending on the audience I don't want to confuse less-technical readers.  I like the consistency of sticking with `%20` and the avoidance of possible confusion between when to encode `+` and when not.  But, y, its definitely not as readable and its the difference between typing 1 char vs 4 (including the shift key).

Reply:
doing the encoding manually is not going to be a great UX regardless so I think our primary audience should be non-technical users and the general recommendation should be to use something to handle the encoding. This is the only such encoding convenience and it's likely to breed confusion, so pursuing consistency for a non-technical audience seems like the best way to head off issues around not understanding the subtleties between `foo+bar`, `foo%20bar`, and `foo%2Bbar`.